src/main/webapp/js/instructorFeedbacks.js

```diff
@@ -184,9 +184,7 @@ function bindCopyButton() {
 }
 
 function bindCopyEvents() {
-
     $('#copyTableModal >tbody>tr').on('click', function(e) {
-        e.preventDefault();
 
         var currentRow = $(this);
```

Comment:
Since this is done in many places throughout the file, we can create an issue for it and change it all together. If I change it only in this function now, it will be inconsistent.